.gitignore

```diff
@@ -145,3 +145,6 @@ cython_debug/
 *.sql
 *.sqlite
 *.xml
+/workflow_api.json
+/workflow_api.py
+/custom_nodes/
```

Comment:
I don't think these are needed in .gitignore. Best practice is to just push one file at a time for now.